### PR TITLE
Added a parameter for greeting message

### PR DESCRIPTION
**Pull request by kkirtivasan**

### CODE REVIEW

File: src/main/java/com/example/Main.java [hunk 1]
```diff
@@ -1,11 +1,14 @@
 package com.example;
 
 public class Main {
+
+    static String greetingMessage = "Hello world!! How are you?";
```
Comment (Argon28) [inline src/main/java/com/example/Main.java]: Wouldn't it be better to have this as a constant?

**Review by Argon28 — CHANGES_REQUESTED** [review]

File: src/main/java/com/example/Main.java (hunk 1 above)
Comment (kkirtivasan) [reply src/main/java/com/example/Main.java]: You're right. I will change it.

Comment (kkirtivasan) [reply src/main/java/com/example/Main.java]: @Argon28 Done.

**Review by Argon28 — APPROVED** [review]
Looks good!

**Pull request merged**